macos/FluentUITestViewControllers/TestColorViewController.swift

```diff
@@ -78,6 +79,19 @@ class TestColorViewController: NSViewController {
 		view = containerView
 	}
 
+	// MARK: ColorProviding Protocol
+
+	var primary: NSColor = useColorProvider ? (NSColor(named: "Colors/DemoPrimaryColor"))! : Colors.Palette.communicationBlue.color
```

Comment:
Should we just make this `var primary: NSColor = NSColor(named: "Colors/DemoPrimaryColor"))!`? If `useColorProvider` is false, won't these not get called?

Reply:
OH, good catch. That was part of an earlier iteration where I was going to try and keep the existing "Default Colors" behavior, which I now believe is overkill since we'll be deprecating this.